hw/net/rocker/rocker_hw.h

```diff
@@ -165,6 +167,14 @@ enum {
     ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_DEL,
     ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_GET_STATS,
 
+    ROCKER_TLV_CMD_TYPE_CLEAR_PORT_STATS,
+    ROCKER_TLV_CMD_TYPE_GET_PORT_STATS,
+
+    ROCKER_TLV_CMD_TYPE_P4_RMT_TABLE_ENTRY_ADD,
```

Comment:
RMT = Reconfigurable Match Table  - This achronym comes from one of the
initial papers on reconfigurable packet processing (I'll send you the exact
name of the paper).
I am using it here as a common identifier that can be shared by P4 worlds
(L2L3 and future). Each P4 world will add its own tables but share a single
TLV type to ADD and table entry.

On Sun, Apr 19, 2015 at 9:09 PM, Scott Feldman <notifications@github.com>
wrote:

> In hw/net/rocker/rocker_hw.h
> <https://github.com/pbhide/qemu-rocker/pull/1#discussion_r28663140>:
>
> > @@ -165,6 +167,14 @@ enum {
> >      ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_DEL,
> >      ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_GET_STATS,
> >
> > +    ROCKER_TLV_CMD_TYPE_CLEAR_PORT_STATS,
> > +    ROCKER_TLV_CMD_TYPE_GET_PORT_STATS,
> > +
> > +    ROCKER_TLV_CMD_TYPE_P4_RMT_TABLE_ENTRY_ADD,
>
> What does the "RMT" part of "P4_RMT" mean?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbhide/qemu-rocker/pull/1/files#r28663140>.
>